src/Modules/UngradedModules.jl

```diff
@@ -1024,11 +1040,11 @@ julia> a2 == b
 true
 ```
 """
-function hom(F::FreeMod, M::ModuleFP{T}, V::Vector{<:ModuleFPElem{T}}) where T
+function hom(F::FreeMod, M::ModuleFP{T}, V::Vector{<:ModuleFPElem{T}}; check::Bool=true) where T
   base_ring(F) === base_ring(M) || return FreeModuleHom(F, M, V, base_ring(M))
   return FreeModuleHom(F, M, V)
 end
```

Comment:
@jankoboehm : I added a dummy `check` flag. It's not doing anything, but it allows to call the hom constructor for any kind of modules with the same signature now. I hope this is ok from your side?

Reply:
What is it doing for other signatures? Does it apply here?